DatadogSessionReplay/Sources/SessionReplay.swift

```diff
@@ -19,20 +19,53 @@ public enum SessionReplay {
     /// - Parameters:
     ///   - configuration: Configuration of the feature.
     ///   - core: The instance of Datadog SDK to enable Session Replay in (global instance by default).
+    ///   - startRecordingImmediately: When `true`, the recording starts automatically; when `false` it doesn't and the recording will need to be started manually (`true` by default).
     public static func enable(
-        with configuration: SessionReplay.Configuration, in core: DatadogCoreProtocol = CoreRegistry.default
+        with configuration: SessionReplay.Configuration,
+        in core: DatadogCoreProtocol = CoreRegistry.default,
+        startRecordingImmediately: Bool = true
```

Comment:
question/ Is there any extra motivation behind introducing a new parameter to `SR.enable()` vs adding it to the existing `SessionReplay.Configuration`? The latter could be more idiomatic choice, no?

In all our modules (RUM | Logs | Trace | ...) we aim at keeping the `enable()` functions lean by separating `configuration:` and `core:` parameters. Here we seem to diverge from existing convention.

Reply:
Not really, I started like this but actually thought of including it in the configuration. I will make the change!

Reply:
Great! Also, the [iOS > Public API Convention in iOS SDK 2.x > Rule 2](https://datadoghq.atlassian.net/wiki/x/6hh2t) makes a good guideline on this element. Keeping `.enable(with:in:)` should be preferred if possible.